docs/implementation/customresources.md

```diff
@@ -0,0 +1,1889 @@
+## Custom Resources
```

Comment:
Use sample instead of .yaml (link to the .yaml like 'click this link to see its implementation...').